### PR TITLE
[ENH] :sparkles: add global research

### PR DESCRIPTION
**Pull request by Donatien26**
Signed-off-by: Donatien ENEMAN <donatien.eneman@insee.fr>

### CODE REVIEW

**Comment by mehdimaaref7** [comment]
Maybe we should abort the modifications on  `.eslintrc` file.

**Review by CChemin — CHANGES_REQUESTED** [review]
when you go from simple search to complex search and then return on simple search the field is disabled

File: src/components/shared/searchFormular/index.tsx [hunk 1]
```diff
@@ -1,73 +1,111 @@
...
 import { useTranslation } from 'react-i18next';
 import { useForms } from 'src/lib/hooks/technics/useForms';
 import { Field } from 'src/lib/model/field';
 import ExpandButton from '../expandButton/expand-button';
-import TextFieldInfo from '../formular/fields/textFieldInfo';
 import GenerateFields from '../formular/fields/utils';
 
 interface props {
 	realm: string;
 	userStorage?: string;
```
Comment (CChemin) [inline src/components/shared/searchFormular/index.tsx:18]: remove this

Comment (CChemin) [reply src/components/shared/searchFormular/index.tsx:18]: and rename props to make sonar happy ^^

File: src/lib/hooks/organization/useGetOrganizations.tsx [hunk 2]
```diff
@@ -29,15 +29,19 @@ export const useGetOrganizations = (realm?: string, userStorage?: string) => {
 
 	const execute = async (
 		realm: string,
-		{ identifiant, mail }: searchRequestOrganization,
+		{ identifiant, mail, typeRecherche }: searchRequestOrganization,
```
Comment (CChemin) [inline src/lib/hooks/organization/useGetOrganizations.tsx:32]: why not getting the entire searchRequestOrganization and then pass the entire searchRequestOrganization

**Review by CChemin — APPROVED** [review]

**Pull request merged**